docs/Theme.md

```diff
@@ -35,78 +35,116 @@ Size values are all provided as Number, and follow the existing sizes used withi
+    - **Default**: "#d4351c"
 
 - **TitleSize** - Size of the font used for the component title
     - **Type**: Size
+    - **Default**: 32
 
 - **Size** - Size of the font used for all text apart from the component title
     - **Type**: Size
+    - **Default**: 16
 
 - **Font** - Font used for the text of the component. This can be any supported font within Power Apps, e.g. "Arial", "Times New Roman" etc.
     - **Type**: Text
+    - **Default**: "Arial"
 
 - **TitleColor** - Color of the text in the component title
     - **Type**: Color
+    - **Default**: "#0b0c0c"
 
 - **Color** - Color of all text apart from the component title and error message
     - **Type**: Color
+    - **Default**: "#0b0c0c"
 
 - **InputBorderThickness** - Thickness of the border around input controls
     - **Type**: Size
+    - **Default**: 2
 
 - **InputBorderRadiusTopLeft** - Radius of the border of the top left corner of input controls. 0 is a square corner, increase the value to increase the roundness
     - **Type**: Size
+    - **Default**: 0
 
 - **InputBorderRadiusTopRight** - Radius of the border of the top right corner of input controls. 0 is a square corner, increase the value to increase the roundness
     - **Type**: Size
+    - **Default**: 0
 
 - **InputBorderRadiusBottomLeft** - Radius of the border of the bottom left corner of input controls. 0 is a square corner, increase the value to increase the roundness
     - **Type**: Size
+    - **Default**: 0
 
 - **InputBorderRadiusBottomRight** - Radius of the border of the bottom right corner of input controls. 0 is a square corner, increase the value to increase the roundness
     - **Type**: Size
+    - **Default**: 0
 
 - **InputBorderColor** - Color of the border around input controls
     - **Type**: Color
+    - **Default**: "#0b0c0c"
 
 - **InputFill** - Color of the background of input controls
     - **Type**: Color
+    - **Default**: "#ffffff"
 
 - **InputPressedBorderColor** - Color of the border around input controls when the input is pressed
     - **Type**: Color
+    - **Default**: "#0b0c0c"
 
 - **InputPressedColor** - Color of the text in input controls when the input is pressed
     - **Type**: Color
+    - **Default**: "#0b0c0c"
 
 - **InputPressedFill** - Color of the background of input controls when the input is pressed
     - **Type**: Color
+    - **Default**: "#ffffff"
 
 - **InputHoverBorderColor** - Color of the border around input controls when the input is hovered over
     - **Type**: Color
+    - **Default**: "#0b0c0c"
 
 - **InputHoverColor** - Color of the text in input controls when the input is hovered over
     - **Type**: Color
+    - **Default**: "#0b0c0c"
 
 - **InputHoverFill** - Color of the background of input controls when the input is hovered over
     - **Type**: Color
+    - **Default**: "#ffffff"
+
+- **InputDisabledBorderColor** - Color of the border around input controls when the component is in disabled mode
+    - **Type**: Color
```

Comment:
We call these 'input controls' here but for error border we say 'input fields'. I think input controls seems to be most popular so change to this where appropriate